src/main/kotlin/utilities/StringBasedCalculator.kt

```diff
@@ -0,0 +1,57 @@
+package utilities
+
+import constant.StringConstant
+import enums.Operators
+import enums.Operators.*
+
+class StringBasedCalculator(
+    expression: String?,
+    var numbers: List<String> = mutableListOf(),
+    var operators: List<String> = mutableListOf()
+) {
+
+    init {
+        if (expression.isNullOrBlank()) {
+            throw IllegalArgumentException("입력값은 null 이거나 공백 문자일 수 없습니다.")
+        }
```

Comment:
코틀린에서는 예외를 활용한 코드의 제한이 가능합니다. 
```suggestion
       require (!expression.isNullOrBlank()) {
            throw IllegalArgumentException("입력값은 null 이거나 공백 문자일 수 없습니다.")
        }
```

Reply:
require구문이 있군요! 활용해보겠습니다